app/controllers/items_controller.rb

```diff
@@ -4,9 +4,13 @@ def new
     type = params[:type]
     druid = params[:druid]
     targets = params[:targets]
+    raise 'missing required params' unless type && druid && targets
```

Comment:
This is a pretty useful overview: http://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters